### ISSUE TITLE
MapSettingsControl does not update

### ISSUE THREAD

**Issue by deeplook** [heremaps/here-map-widget-for-jupyter#8]
After creating/showing a `MapSettingsControl` it seems impossible to add layers to it such that they appear in the control. It would be nice to see that happen like in `ipyleaflet.LeayersControl`. 

```python
import os
from here_map_widget import (Map, DefaultLayers, DefaultLayerNames,
    GeoJSON, MapSettingsControl, ZoomControl)

api_key = os.getenv("HEREMAPS_API_KEY")
basemap = DefaultLayers(layer_name=DefaultLayerNames.raster.normal.map)
m = Map(center=[52.5, 13.4], zoom=10, basemap=basemap, api_key=api_key) 
m.add_control(ZoomControl(alignment="LEFT_TOP"))
basemaps = [DefaultLayerNames.raster.normal.map, DefaultLayerNames.raster.normal.transit]
url = "https://raw.githubusercontent.com/johan/world.geo.json/master/countries.geo.json"
layers = [GeoJSON(url=url)]
layers = [{"label": "countries", "layer": GeoJSON(url=url)}]
msc = MapSettingsControl(basemaps=basemaps, layers=layers)

# does not add second layer
msc.layers.append({"label": "countries2", "layer": GeoJSON(url=url)})

m.add_control(msc)

# does not add second layer
msc.layers.append({"label": "countries2", "layer": GeoJSON(url=url)})

m
```

**Comment by sackh** [comment]
Thanks, @deeplook for opening this issue, this will be fixed in the upcoming release.

**Comment by sackh** [comment]
I have added a fix in #16, now you can change values of `basemaps` and `layers` traits post creation.
